.storybook/components/Resizer/Resizer.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import PropTypes from "prop-types";
+import * as sc from "./Resizer.styles";
+
+const propTypes = {
+  children: PropTypes.node,
+  initHeight: PropTypes.number,
+  initWidth: PropTypes.number,
+  onResizeStop: PropTypes.func,
+};
+
+const defaultProps = {
+  children: null,
+  initHeight: 360,
```

Comment:
for another iteration, seems like would be beneficial to have the viewport size instead of random values ... :)